packages/components/tests/integration/components/hds/table/th-sort-test.js

```diff
@@ -14,31 +14,44 @@ module('Integration | Component | hds/table/th-sort', function (hooks) {
 
   test('it renders', async function (assert) {
     await render(hbs`<Hds::Table::ThSort
-  @sortBy='artist'
-  @sortOrder='asc'
   @setSortBy={{this.setSortBy}}
 >artist</Hds::Table::ThSort>`);
 
     assert.dom(this.element).hasText('artist');
   });
+  test('it has the scope attribute, and it is set to column', async function (assert) {
+    await render(hbs`<Hds::Table::ThSort
+  @setSortBy={{this.setSortBy}}
+>artist</Hds::Table::ThSort>`);
+
+    assert.dom('[data-test-table-th-sort]').hasAttribute('scope', 'col');
```

Comment:
[nitpick] for the tests of `Td` we target the element using the ID, for this one the data attribute. We should stick to one (probably the ID, for consistency with all the other components' tests)

Reply:
in some instances there will be an id and some will use a data-test attribute- it depends on whether or not the component supports splattributes, I think.

Reply:
I mean in the tests themselves. why use different approaches? what is the benefit in that case?

Reply:
OK, I see; the `Th-Sort` doesn't have splattributes. But at this point, why not add the `...attributes` also to the `Th-Sort` (even if it's not used directly when it's in a loop)? Would allow us to avoid exceptions, make it more similar to the `Th` and (bonus) make it potentially ready to be used for custom-built sortable table if needed.

Reply:
I don't think there's a way for them to invoke `...attributes` here and `data-test-*` attributes are stripped out for production. It's a legitimate use case and it's reasonable to leave it be.

Reply:
@MelSumner we're already had this conversation multiple times: is common practice in out components to **not** add `data-test` attributes in the components. If you do a full search in the codebase (`/packages/components/addon/components/hds`) you will see that the only place where you find `data-test` attributes in the components it's in the new Table components.

Reply:
I've updated all of these except the th-sort, that will need to stay the exception for now. If you disagree, let's set up a pairing time and you can show me how you would invoke `...attributes`. Otherwise consider this resolved, but will bring it up for future team discussion in general.